### DISCUSSION TITLE
Trouble with Arc Dimensions and unknown attribute 'defpoint2'

### DISCUSSION THREAD

**Discussion by ChaseCrossley** [mozman/ezdxf#505] [Issues]
I am relatively new to DXF files and ezdxf, so please forgive my ignorance if this has been addressed before. I looked on various platforms to see if someone had a similar issue with Arc Dimensions, but I could not find anything related to the issue I was experiencing, so I assumed it was a very niche case. 

I was having trouble saving a document with an Arc Dimension present within the DXF file. I have attached a testing file with which I was having issues. 

I was receiving this error 
```
.../ezdxf/entities/dxfns.py", line 344, in _export_dxf_attribute_optional
    raise const.DXFAttributeError(
ezdxf.lldxf.const.DXFAttributeError: Invalid DXF attribute "defpoint2" for entity ARC_DIMENSION
```
from the following code block
```python3
import ezdxf
doc = ezdxf.readzip('arc-dim.dxf.zip')
doc.saveas('test.dxf')
```
After investigating that a debugger with the specific entity causing the error, I realized there was no attribute called `defpoint2` present within the entity.  I looked under `DXFATTRIBS`. 

I believe I am missing a critical piece of information to understanding what is happening.  Thank you for any help!

testing file: [arc-dim.dxf.zip](https://github.com/mozman/ezdxf/files/6863902/arc-dim.dxf.zip)

**Comment by mozman** [comment]
Created issue #506
